camel/functions/search_functions.py

```diff
@@ -55,6 +55,68 @@ def search_wiki(entity: str) -> str:
     return result
 
 
+def search_duckduckgo(query: str) -> List[Dict[str, Any]]:
+    r"""Use DuckDuckGo search engine to search information for the given query.
+
+    This function queries the DuckDuckGo API for related topics to the given
+    search term. The results are formatted into a list of dictionaries, each
+    representing a search result.
+
+    Args:
+        query (str): The query to be searched.
+
+    Returns:
+        List[Dict[str, Any]]: A list of dictionaries where each dictionary
+        represents a search result.
+            Each dictionary contains the following keys:
+            - 'result_id': A sequential number starting from 1.
+            - 'title': The title or main text of the search result.
+            - 'description': A brief description of the result, often similar
+                to the title.
+            - 'long_description': More detail of the result, often the same as
+                the description.
+            - 'url': The URL of the result.
+
+            Example:
+            {
+                'result_id': 1,
+                'title': 'DuckDuckGo',
+                'description': 'DuckDuckGo is an Internet search engine ' + \
+                    'that emphasizes protecting searchers' privacy.',
+                'long_description': 'DuckDuckGo is an Internet search ' + \
+                    'engine that emphasizes protecting searchers' privacy.',
+                'url': 'https://www.duckduckgo.com'
+            }
+        title, description, url of a search result.
+    """
+    from duckduckgo_search import DDGS
+
+    ddgs = DDGS()
+    max_results = 10
+    results = [r for r in ddgs.text(keywords=query, max_results=max_results)]
+
+    responses = []
+
+    # Iterate over 10 results found
+    for i, result in enumerate(results, start=1):
+        # Extracting data from each result
+        title = result["title"]
+        body = result["body"]
+        url = result["href"]
+
+        # Creating a response object with a similar structure
+        response = {
+            "result_id": i,
+            "title": title,
+            "description": body,
+            "long_description": body,
```

Comment:
why both description and long_description are using body here? how about remove the duplicates